styles/youtube/catppuccin.user.css

```diff
@@ -472,6 +472,17 @@
       --ytd-author-comment-badge-icon-color: @text !important;
     }
 
+    /* youtube live ring */
+    .yt-spec-avatar-shape__badge-text {
+      color: @crust;
+    }
+    .yt-spec-avatar-shape--live-ring {
+      border-color: @accent-color;
+    }
+    .yt-spec-avatar-shape__live-badge {
+      background-color: @accent-color;
+    }
```

Comment:
Can we put this somewhere more relevant, like near the other live/premiere badges/stuff is?